src/gleam_stdlib.js

```diff
@@ -258,3 +258,67 @@ export function regex_scan(regex, string) {
   });
   return List.fromArray(matches);
 }
+
+export function new_map() {
+  return new Map();
+}
+
+export function map_size(map) {
+  return map.size;
+}
+
+export function map_to_list(map) {
+  const result = [];
+  map.forEach(([a, b]) => result.push([JSON.parse(a), b]))
```

Comment:
I think we can do this with the latest unreleased Gleam

```suggestion
  List.fromArray([...map.values()])
```

Assuming the values are now `[originalKey, value]`